README.md

```diff
@@ -42,9 +42,7 @@ You can configure the following services on an existing edgegateway using
 - firewall_service
 - nat_service
 - load_balancer_service
-
-NB: DHCP and VPN Services are not yet supported by the Fog platform underneath.
```

Comment:
If DHCP is still unsupported, we should leave this message intact, excluding the bit about the VPN.